source/code/source/playfab/PlayFabSettings.cpp.ejs

```diff
@@ -5,6 +5,7 @@
 namespace PlayFab
 {
     // Control whether all callbacks are threaded or whether the user manually controls callback timing from their main-thread
+    // Note ANY api call may synchronously throw an exception if the title id is not set, regardless of this flag
```

Comment:
"regardless of this flag" doesn't seem relevant anymore since throwing an exception doesn't violate what this flag represents

Reply:
i.e. throwing an exception isn't a callback so regardless of what this flag is set as, it should always be expected that calling an API method can throw since none of these are marked `noexcept`